src/madhuraMJun20/scannerClass/FibonacciNumbers.java

```diff
@@ -0,0 +1,29 @@
+package madhuraMJun20.scannerClass;
+
+/*Program 2: To print first n Fibonacci Numbers.
+Input n = 6
+Output = 1 1 2  3  5  8
+
+Input: n = 10 
+Output :  1 1 2  3  5  8 13  21 34 55*/
+public class FibonacciNumbers {
+	void findFibonacciNumber(int n) {
+		int firstNum = 0;
+		int nextNum = 1;
+		System.out.print("\nFirst " + n + " Fibonacci numbers :- \n");
+
+		for (int index = 0; index < n; index++) {
+			int sum = firstNum + nextNum;
```

Comment:
Code logic is good. I suggest you can reduce the code without using the extra variable "sum" here. Think about it.

Reply:
Ok. I will try